sk-csharp-chatgpt-plugin/azure-function/sk-chatgpt-azure-function.csproj

```diff
@@ -0,0 +1,58 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <RepoRoot>$([System.IO.Path]::GetDirectoryName($([MSBuild]::GetPathOfFileAbove('.gitignore', '$(MSBuildThisFileDirectory)'))))</RepoRoot>
+    <UserSecretsId>b4703849-9b3a-41c3-b1bb-5ede533963c4</UserSecretsId>
```

Comment:
This is the same value as used in sk-csharp-azure-functions, is this intentional or should you generate a unique value?

Reply:
Changed to a different GUID.